cmd/juju/model/defaultscommand.go

```diff
@@ -97,120 +131,425 @@ func (c *defaultsCommand) SetFlags(f *gnuflag.FlagSet) {
 		"json":    cmd.FormatJson,
 		"tabular": formatDefaultConfigTabular,
 	})
-	f.BoolVar(&c.reset, "reset", false, "Reset the provided keys to be empty")
+	f.StringVar(&c.reset, "reset", "", "Reset the provided keys")
```

Comment:
Fuuuuuuuuuugde. I didn't know our cmd framework and flag add-on let us do this. Thanks for the heads up.